scipy/linalg/tests/test_lapack.py

```diff
@@ -1645,3 +1645,128 @@ def test_getc2_gesc2():
         else:
             assert_array_almost_equal(desired_cplx.astype(dtype),
                                       x/scale, decimal=4)
+
+    
+@pytest.mark.parametrize("dtype", DTYPES)
+@pytest.mark.filterwarnings("ignore:Casting complex")
+def test_gttrf_gttrs(dtype):
+
+    np.random.seed(42)
+    # use ?gttrf and ?gttrs to solve a random linear system for each dtype
+    n = 4
+    # set appropriate test tolerance based on the data type
+    rtol = 250*np.finfo(dtype).eps
```

Comment:
Let's put comment above inline
```suggestion
    rtol = 250*np.finfo(dtype).eps  # set test tolerance appropriate for data type
```

Reply:
👍